cpp/src/io/orc/timezone.cpp

```diff
@@ -13,71 +13,205 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-#include "timezone.h"
+#include "timezone.cuh"
+
 #include <algorithm>
 #include <fstream>
 
 namespace cudf {
 namespace io {
-// NOTE: Assumes little-endian platform
-#ifdef _MSC_VER
-#define bswap_32(v) _byteswap_ulong(v)
-#define bswap_64(v) _byteswap_uint64(v)
-#else
-#define bswap_32(v) __builtin_bswap32(v)
-#define bswap_64(v) __builtin_bswap64(v)
-#endif
 
 #define TZIF_MAGIC (('T' << 0) | ('Z' << 8) | ('i' << 16) | ('f' << 24))
```

Comment:
I was planning to exclude these from the PR since @kaatish already has code changes that cover this, but I might be easier for him to merge if I just make the change here.